guide/start/running.md

```diff
@@ -68,7 +69,24 @@ $ yum install apache-brooklyn-{{site.brooklyn-version}}-1.rpm
 </div>
 <div id="impl-3" class="tab-pane fade">
 
-<strong class="hidden started-pdf-include">c) OSX / Linux</strong>
+<strong class="hidden started-pdf-include">c) Debian / Ubuntu</strong>
+
+For Ubuntu and Debian users, the recommended way to install Apache Brooklyn is to use the deb file. 
+
+The deb file is the de facto standard for packaging software on these Linux distributions and provides a mechanism for installing, upgrading and removing packages such as Apache Brooklyn. The deb package contains all the necessary files associated with the Apache Brooklyn application. 
+
+Download the Apache Brooklyn [deb distribution](https://www.apache.org/dyn/closer.lua/brooklyn/apache-brooklyn_{{site.brooklyn-version}}_noarch.deb){:target="_blank"}.
+
+Once downloaded, run the following shell command:
```

Comment:
Having the java package installed (say openjdk-7-jre) is not enough unfortunately, dpkg will fail without adding `default-jre-headless`.